source/operator.rst

```diff
@@ -64,6 +69,58 @@ main subsystems of the cluster:
      - ``my-cluster-ssl-internal``
      - A secret with TLS certificate generated for *internal* communications, see :ref:`tls` for details
 
+   * - updateStrategy
+     - string
+     - ``SmartUpdate``
+     - A strategy Operator uses for :ref:`upgrades<operator-update>`
+
+.. _operator.upgradeoptions-section:
+
+`Upgrade Options Section <operator.html#operator-upgradeoptions-section>`_
+--------------------------------------------------------------------------------
+
+The ``upgradeOptions`` section in the `deploy/cr.yaml <https://github.com/percona/percona-xtradb-cluster-operator/blob/master/deploy/cr.yaml>`_ file contains various configuration options to control Percona XtraDB Cluster upgrades.
+
+.. tabularcolumns:: |p{2cm}|p{13.6cm}|
+
++-----------------+-------------------------------------------------------------------------------------------+
+|                 | .. _upgradeoptions-versionserviceendpoint:                                                |
+|                 |                                                                                           |
+| **Key**         | `upgradeOptions.versionServiceEndpoint                                                    |
+|                 | <operator.html#upgradeoptions-versionserviceendpoint>`_                                   |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Value**       | string                                                                                    |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Example**     | ``https://check.percona.com/versions``                                                   |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Description** | The Version Service URL used to check versions compatibility for upgrade                  |
++-----------------+-------------------------------------------------------------------------------------------+
+|                                                                                                             |
++-----------------+-------------------------------------------------------------------------------------------+
+|                 | .. _upgradeoptions-apply:                                                                 |
```

Comment:
Can we enumerate the valid values?  This should be set to "never" or "disabled" to completely disable automatic upgrades, otherwise it can be set to "latest" or "recommended" or to a specific version string of PXC that is wished to be version-locked (so that the user can control the version running, but use automatic upgrades to move between them).